protocol-units/da/m1/util/src/config/local/m1_da_light_node.rs

```diff
@@ -1,13 +1,90 @@
 use crate::config::common::{
 	default_celestia_rpc_connection_hostname, default_celestia_rpc_connection_port,
 	default_celestia_websocket_connection_hostname, default_celestia_websocket_connection_port,
-	default_da_signers_sec1_keys, default_da_signing_private_key,
 	default_m1_da_light_node_connection_hostname, default_m1_da_light_node_connection_port,
 	default_m1_da_light_node_listen_hostname, default_m1_da_light_node_listen_port,
 };
+use ecdsa::SigningKey;
+use k256::Secp256k1;
 use serde::{Deserialize, Serialize};
 use std::collections::HashSet;
 
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
+pub struct DaSigners {
+	pub da_signing_private_key_hex: String,
+	pub da_signers_public_keys_hex: HashSet<String>,
```

Comment:
These names are quite a handful to type if they are meant to be used in JSON and similar schemas. Why do we need to nag the user every time that this is hex, when it should be documented (in the doc comments for this structure and in the example config)?

Reply:
The containing structure does not have `#[serde(flatten)]` as a field in `Config`, so
the full field name would be resolved like `da_signers.da_signers_public_keys_hex`. I think it's poor UX. Plug for more discussion in #533.